smrf/envphys/storms.py

```diff
@@ -2,69 +2,67 @@
 import pandas as pd
 
 
-def time_since_storm(precipitation, perc_snow, time_step=1/24, mass=1.0,
-                     time=4, stormDays=None, stormPrecip=None, ps_thresh=0.5):
+def time_since_storm(
+    precipitation,
+    percent_snow_precipitation,
+    storm_days,
+    storm_precip,
+    time_step,
+    mass_threshold=1.0,
+    percent_snow_threshold=0.5
+):
     """
-    Calculate the decimal days since the last storm given a precip time series,
-    percent snow, mass threshold, and time threshold
+    Increase or reset the given storm days and storm precipitation based of
+    given percent snow and mass threshold.
 
-     - Will look for pixels where perc_snow > 50% as storm locations
-     - A new storm will start if the mass at the pixel has exceeded the mass
-         limit, this ensures that the enough has accumulated
+    Steps:
+     - Look for pixels where percent snow precipitation is above the threshold.
+     - Reset storm days if the precipitation at the pixel is above the mass
+         threshold or increase the counter if it is below.
+     - Add the precipitation amount to the storm precipitation.
+
+    *NOTE*: Storm day precipitation is initialized and set to 0 with each
+        processed day. Hence, no cross-day storm tracking is possible if the
+        period between the days falls below the threshold.
 
     Args:
         precipitation: Precipitation values
-        perc_snow: Percent of precipitation that was snow
+        percent_snow_precipitation: Percent of precipitation that was snow
+        storm_days: Storm days to keep track of
+        storm_precip: Keeps track of the total storm precip
         time_step: Step in days of the model run
-        mass: Threshold for the mass to start a new storm
-        time: Threshold for the time to start a new storm
-        stormDays: If specified, this is the output from a previous run of
-            storms else it will be set to the date_time value
-        stormPrecip: Keeps track of the total storm precip
+        (Optional)
+        mass_threshold: Minimum amount of precipitation required to be a storm
+                        (snow mass). Default: 0.5
+        percent_snow_threshold: Minimum percent snow within a pixel in
```

Comment:
I think this is the percent of precip that is _snow_, to account for mixed phase precip. This reads like a spatial metric to me.

Reply:
This is the `ps_thresh` parameter from the initial method signature and the threshold marks the value that has to be exceeded for a pixel to be considered as snowfall. I think we agree on that. 

Can you expand on your comment a bit or suggest a different description.

Reply:
I suggest: _percent_snow_threshold: Minimum percent of precip that is snow within a pixel in fractional amount. Default: 0.5 (50%)_

But this is subjective—perhaps just related to how I read and think 😁. Your choice!